src/Components/Blazor/Directory.Build.props

```diff
@@ -6,5 +6,8 @@
     <VersionPrefix>3.1.0</VersionPrefix>
     <PreReleaseVersionLabel>preview4</PreReleaseVersionLabel>
     <DotNetFinalVersionKind />
+    <!-- This property points to the latest released Microsoft.AspNetCore.App version it needs to be updated to
+         target the latest patch before a preview release. -->
+    <LatestAspNetCoreReferenceVersion>3.1.0</LatestAspNetCoreReferenceVersion>
```

Comment:
Would it make sense to define this like the following?

```
<LatestAspNetCoreReferenceVersion>$(MicrosoftNETCoreAppRefPackageVersion)</LatestAspNetCoreReferenceVersion>
```

... or are there cases where this might not match up?

Reply:
I'm not sure about it, its one of the follow-ups I intend to do with the build folks next week.